tests/global-teardown.ts

```diff
@@ -0,0 +1,5 @@
+const { teardown: teardownDevServer } = require('jest-dev-server');
+
+module.exports = async function globalTeardown() {
+  await teardownDevServer();
+};
```

Comment:
as this is .ts, why do we ened to use commonjs exports?

Reply:
resolved